robottelo.properties.sample

```diff
@@ -72,3 +72,62 @@ driver=firefox
 # URL of the  exported data archive (typically a .tgz containing a bunch of CSV
 # files together with repo data)
 #export_tar.url=http://example.org/sat5_export_data.tgz
+
+# Section for performance tests parameters.
+[performance]
+# Control whether or not to time on hammer commands in robottelo/cli/base.py
+# Default set to be 0, i.e. no timing of performance is measured and thus no
+# interference to original robottelo tests.
+test.foreman.perf=0
+# Folowing entries are used for preparation of performance tests after a fresh install.
+# They will be used by test/foreman/performance/test_standard_prep.py, which supports:
+# 1) downloading manifest,
+# 2) uploading manifest to subscription,
+# 3) updating Red Hat CDN URL,
+# 4) enabling key repositories: rhel6-rpms, rhel7-rpms, rhel6-kickstart-rpms, rhel7-kickstart-rpms,
+#    rhel6-optional-rpms, rhel7-optional-rpms, rhel6-optional-source-rpms, rhel7-optional-source-rpms,
+#    rhel6-optional-debug-rpms, r7-optional-debug-rpms
+# Note that this preparation step is not required as long as satellite server is already configured.
+test.manifest.location=http://path/to/manifests/
+test.manifest.file=example-manifest-name.zip
+test.cdn.address=http://example.redhat.cdn/pub
+# Product-ID for enabling repositories by robottelo.cli.repository_set.RepositorySet
+test.pid=15
+# Set variables for a list of virtual machines' IP address, which would conduct concurrent system 
+# subscriptions.
+# Assume that those vms are pre-provisioned.
+# As for current workaround, append each IP address to the end, join by "," without any spaces.
+test.virtual_machines_list=127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,
+
+# Default Organization ID as option of various hammer commands.
+test.organization.id=1
```

Comment:
@Ichimonji10 The point is that, for performance tests, we just want to use the default organization. And I tested on two builds of Satellite 6, the default org id would be both "1". So that's why I just set a 1 here across all test cases.